src/cmd/vm.rs

```diff
@@ -364,7 +372,7 @@ fn get_cheeps_image_name(config: &Config, is_container: bool, branch: &str) -> R
     Ok(cheeps)
 }
 
-fn run_acloudw_cmd(opts: &[&str]) -> Result<()> {
+fn run_acloudw_cmd(opts: &[&str]) -> Result<String> {
```

Comment:
Port number is a 16-bit unsigned number, so the return value type should be u16, not a String.

Reply:
Indeed, I updated it.